src/facilities-context/reducer.tsx

```diff
@@ -60,11 +63,14 @@ export function facilitiesReducer(
       return { ...state, referenceFacilities: action.payload };
 
     case actions.CLEAR_REFERENCE_FACILITIES:
-      return { ...state, referenceFacilities: {} };
+      return { ...state, referenceFacilities: {}, canUseReferenceData: false };
 
     case actions.CLEAR_FACILITIES:
       return { ...state, facilities: {}, loading: false };
```

Comment:
Would we need to dispatch `type: SET_CAN_USE_REFERENCE_DATA, payload: false` from the `initializeFacilities` else branch, so that it is set to false whenever facilities or referenceFacilities are cleared?

Reply:
yeah that would probably be better than doing it this way ... more explicit, probably more reliable